apps/web/src/app/(default)/results/[address]/page.tsx

```diff
@@ -3,9 +3,19 @@
 import { notFound, useParams } from "next/navigation";
 import MevAttackResults from "@/components/MevAttackResults";
 import { isValidSolanaAddress } from "@/utils/blockchain";
+import useLocalStorage from "@/hooks/useLocalStorage";
+import { LocalStorage } from "@/constants/storage";
+import { useEffect } from "react";
 
 export default function Page() {
   const params = useParams<{ address: string }>();
+  const [, setLookupAddress] = useLocalStorage<string | null>(LocalStorage.LOOKUP_ADDRESS, null);
+
+  useEffect(() => {
```

Comment:
This component will re-render whenever `params` changes, so we don't need useEffect here, can we modify this to be inline without useEffect?

Happy to go over this in more detail if it would be useful!